bundler/lib/bundler/templates/newgem/README.md.tt

```diff
@@ -1,18 +1,20 @@
 # <%= config[:constant_name] %>
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/<%= config[:namespaced_path] %>`. To experiment with that code, run `bin/console` for an interactive prompt.
+TODO: Delete this and the text below, and describe your gem
 
-TODO: Delete this and the text above, and describe your gem
+Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/<%= config[:namespaced_path] %>`. To experiment with that code, run `bin/console` for an interactive prompt.
 
 ## Installation
 
+TODO: Replace the `UPDATE_WITH_YOUR_GEM_NAME_PRIOR_TO_RELEASE_TO_RUBYGEMS_ORG` with your gem name before releasing it to RubyGems.org or delete this section if you do not plan to release to RubyGems.org. Please do not do it earlier due to security reasons.
```

Comment:
> The security concern is when changing the gem name, not when deleting the section.

Not exactly. Even if you do not release someone can release it for you and brandjack. Which was happening.

>  I also think we should link somewhere to try explain what the security concern is, or explain it ourselves. I

Fair point. Do you want me to write a blog post for https://blog.rubygems.org/ about this?

Reply:
Also, isn't the safe path the exact opposite to what we are recommending? 

> Replace the `UPDATE_WITH_YOUR_GEM_NAME_PRIOR_TO_RELEASE_TO_RUBYGEMS_ORG` with your gem name before releasing it to RubyGems.org 

Shouldn't it be **after** rather than **before**, or maybe **right after** if we want to recommend doing it at that same-ish time?

Reply:
> Not exactly. Even if you do not release someone can release it for you and brandjack. Which was happening.

Right, I meant it regarding the trailing "Please do not do it earlier due to security reasons" sentence. In the case of deleting the section because of not planning to release to rubygems.org, the timing doesn't really matter, no?

Reply:
I think a blog post would be nice, but I think @simi has some concerns about whether that would actually give this attack more visibility than we want? I'll let @simi chime in!

Reply:
It looks like `after` was replaced with `before`, but `PRIOR_TO` was left unchanged.

I've opened #6338 to update this.

I also don't know if a blog post ended up being written, but if so, perhaps we can link to it.